modules/renter/renter.go

```diff
@@ -182,8 +182,8 @@ type Renter struct {
 	// accessed in isolation.
 	//
 	// TODO: Currently the download history doesn't include repair-initiated
-	// downloads, and instead only contains user-initiated downlods.
-	downloadHistory   []*download
```

Comment:
I'm not sure if we should change the history to a map. That means you can only have 1 entry for each file even if you download it multiple times.

Reply:
I had done that mostly for the method that removes a particular download so that I didn't have to iterate over the entire slice to remove the download I was looking for.

Do you think that iteration is fine in this case?  Or do you have a better option in mind?

Reply:
Change to a map of slices